### DISCUSSION TITLE
Multi-language UI

### DISCUSSION THREAD

**Discussion by michaelroussel2494** [pymedusa/Medusa#7584] [Ideas & Feature requests]
Hello Teams

I like very much medusa, it have a many good fonctionnality.
It is possible to provide a multi-language UI ?
I am french an it's more easy for my familly witch don't speek anglais very much a can change language of interface.

Thanks in advance.

I have trie to maek a change, but i don't know witch file to editing.

Best regards

**Comment by OmgImAlexis** [comment]
Every single page will need to be updated to support using i18n in Vue. We'd then need to add a API endpoint which returns the language file which would be cached in `localStorage`. It's not going to be an easy thing.

**Comment by ciltocruz** [comment]
What about this?
Sickchill has other UI languages and if Medusa is a fork ...

Thanks!!

**Comment by OmgImAlexis** [comment]
They’re forks but nothing alike code wise. Did you not read my comment?

**Comment by ciltocruz** [comment]
Of course I've read it. 
So as not to be similar in the source code, the interface is 90% the same. That's why I thought it would be possible to add the UI translation in Medusa just as sickchill has it.

At least the ticket is still open. Which means this option has not been rejected. 

Thank you and a greeting.

**Comment by p0psicles** [comment]
They might look alike, but technically very different. We use js framework vue.js for our frontend

**Comment by ciltocruz** [comment]
Look. you see?
I did not know that.

Thanks for the work as well.

A greeting!
